docs/en/ingest-management/fleet/add-fleet-server.asciidoc

```diff
@@ -2,46 +2,113 @@
 = Add a {fleet-server}
 
 To use {fleet} for central management, a <<fleet-server,{fleet-server}>> must
-be running and accessible to your hosts. This page describes how to add a
-{fleet-server} to an {ecloud} or self-managed deployment.
+be running and accessible to your hosts. There are a few approaches you can take
+when deploying {fleet-server}:
+
+* <<deployed-in-cloud>> as part of the hosted {ess}. Elastic manages both {fleet-server} and {es}.
+* <<deployed-on-prem>> to work with {es} running on-premises. You self-manage both {fleet-server} and {es}.
+* <<fleet-server-on-prem-es-cloud>>. You manage {fleet-server} and Elastic manages {es}.
```

Comment:
Let me know what you think!

Reply:
LGTM. We'll see what Nima and dev team says.